Changed the use of vector time, changed from evolveTime to situationDevelopTime, updated documentation

tomarnepedersen opened the pull request:
vectorTime may now be a single float (exact time) or a list of two floats [from, to]. vectorRange is removed from settings file.
evolveTime is changed to situationDevelopTime.

Run tests, check that it generates situations (e.g. using examples files), check documentation.

StephanieKemna reviewed (changes requested):
In general it looks good! :)
However, I found one hiccup: vectorRange is still in the settings file, and when I remove it, I have problems with generating situations, e.g. for baseline_situations_generated

tomarnepedersen commented:
Thanks. Will have a look at it!

tomarnepedersen commented:
Hei og god morgen,

Hvilke hick-ups så du? Når jeg kjører uv run

trafficgen gen-situation -v  -o ./data/test_output

Så blir de 55 situasjonene generert. Gjorde det etter at jeg hadde tatt bort vector_range... (Usikker på hva som har skjedd der, for mener jeg hadde tatt den bort allerede 😕)

TA


Best regards

for DNV AS

Tom Arne Pedersen, PhD

Principal Researcher - Ship Autonomy
Group Research & Development



DNV AS, Norway
***@***.******@***.***>
Mobile +4795280695  |  Direct +4795280695

dnv.com<https://www.dnv.com/>  |  LinkedIn<https://www.linkedin.com/company/4411/>





[DNV email logo]<https://www.dnv.com/>



________________________________
From: Stephanie Kemna ***@***.***>
Sent: Thursday, May 8, 2025 16:36
To: dnv-opensource/ship-traffic-generator ***@***.***>
Cc: Pedersen, Tom Arne ***@***.***>; Author ***@***.***>
Subject: Re: [dnv-opensource/ship-traffic-generator] Changed the use of vector time, changed from evolveTime to situationDevelopTime, updated documentation (PR #63)



Caution: This email originated from outside of the DNV Organization. Do not click links or open attachments unless you recognize the sender and know the content is safe.



@StephanieKemna requested changes on this pull request.

In general it looks good! :)
However, I found one hiccup: vectorRange is still in the settings file, and when I remove it, I have problems with generating situations, e.g. for baseline_situations_generated

—
Reply to this email directly, view it on GitHub<https://github.com/dnv-opensource/ship-traffic-generator/pull/63#pullrequestreview-2825365514>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AT7NMNHNAGAGCRR3FDVX7RT25NTW7AVCNFSM6AAAAAB4DBHIUGVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDQMRVGM3DKNJRGQ>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

**************************************************************************************
This e-mail and any attachments thereto may contain confidential information and/or information protected by intellectual property rights for the exclusive attention of the intended addressees named above. If you have received this transmission in error, please immediately notify the sender by return e-mail and delete this message and its attachments. Unauthorized use, copying or further full or partial distribution of this e-mail or its contents is prohibited.
**************************************************************************************

StephanieKemna commented:
I tried with
> trafficgen gen-situation -s .\data\baseline_situations_generated -o .\data\test_output2 -v 
after activating the .venv and reinstalling the package.
Will send you the output over chat.
